Document/0x04b-Mobile-App-Security-Testing.md

```diff
@@ -2,22 +2,25 @@
 * **Network:** Usage of insecure or unencrypted communication channel, missing SSL certificate pinning etc.
 * **Remote Services:** Flawed authentication and session management, vulnerable server side functions etc.
 
--- TODO --
+The following sections will show how to use the OWASP mobile application security checklist and testing guide during a security test. It is split into four sections:
 
-The following sections will show how to use the OWASP mobile application security checklist and testing guide during a security test.
+* **Preparation** - defining the scope of security testing, such as which security controls are applicable, what goals the development team/organization have for the testing, and what counts as sensitive data in the context of the test. 
+* **Intelligence Gathering** - involves analysing the **environmental** and **architectural** context of the app, to gain a general contextual understanding of the app.
+* **Threat Modelling** - consumes information gathered during the earlier phases to determine what threats are the most likely, or the most serious, and therefore which should receive the most attention from a security tester. Produces test cases that may be used during test execution.
+* **Vulnerability Analysis** - identifies vulnerabilities using the previously created test cases, including static, dynamic and forensic methodologies.
 
-### Preparation - Defining The Baseline
+----------
 
-First of all, you need to decide what security level of the [MASVS](https://github.com/OWASP/owasp-masvs) to test against. The security requirements should ideally have been decided at the beginning of the SDLC - but unfortunately we do not live in an ideal world. 
+### Preparation
 
-Different organizations have different security needs, and different amounts of resources to invest in test activity. While the controls in MASVS Level 1 (L1) are applicable to all mobile apps, it is a good idea to walk through the entire checklist of L1 and Level 2 (L2) MASVS controls with technical and business stakeholders to agree an appropriate level of test coverage.
+Before conducting a test, an agreement must be reached as to what security level of the [MASVS](https://github.com/OWASP/owasp-masvs) to test against. The security requirements should ideally have been decided at the beginning of the SDLC, but this may not always be the case. In addition, different organizations have different security needs, and different amounts of resources to invest in test activity. While the controls in MASVS Level 1 (L1) are applicable to all mobile apps, it is a good idea to walk through the entire checklist of L1 and Level 2 (L2) MASVS controls with technical and business stakeholders to agree an appropriate level of test coverage.
```

Comment:
Can you please add a subsection "#### References" at the very end of this document? Then add the link for the MASVS like this to the sub-section:

#### References
[1] MASVS - https://github.com/OWASP/owasp-masvs

Delete the link from the text and replace it with this: "MASVS<sup>[1]</sup>"

See also https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md#external-references